### PR TITLE
#5677 [Manager] Ensure only one cypress version is used in our deps

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist

- [x] [Review Contribution Guidelines](https://oharastream.github.io/en/docs/master/contributing/)
- [x] Add at least one reviewer (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add an assignee (of course, you are the assignee by default)
- [x] Do not set milestone, projects and labels (PR labels are added automatically by Jenkins)
- [x] Link to an issue
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (If the failed tests are unrelated to this PR, add a comment in the PR thread)

### CODE REVIEW

**Comment by konekoya** [comment]
retry manager-e2e

File: ohara-manager/client/package.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
     "@material-ui/core": "4.11.0",
     "@material-ui/icons": "4.9.1",
     "@material-ui/lab": "4.0.0-alpha.56",
-    "@testing-library/cypress": "6.0.0",
+    "@testing-library/cypress": "7.0.1",
```
Comment (saivirtue) [inline ohara-manager/client/package.json:43]: Hmm...so we should follow the latest version of this library?

Comment (konekoya) [reply ohara-manager/client/package.json:43]: Yes, it was fixed in 7.x

Comment (konekoya) [reply ohara-manager/client/package.json:43]: https://github.com/testing-library/cypress-testing-library/issues/98#issuecomment-579371595

**Comment by konekoya** [comment]
retry

**Comment by konekoya** [comment]
Wrong branch, i will open a new one

**Pull request closed without merging**